backend/cleanup_repo.py

```
@@ -0,0 +1,179 @@
+        if 'combined_tickers' not in csv_file:  # Keep fresh ticker lists
+            size = get_file_size(csv_file)
+            bloat_files.append((csv_file, size, 'test_csv'))
+
+    # 2. Log files
+    log_files = glob.glob('*.log')
+    for log_file in log_files:
+        size = get_file_size(log_file)
+        bloat_files.append((log_file, size, 'log'))
+
+    # 3. Python cache files
+    pycache_dirs = []
```

Ruff (0.14.3)

Local variable `pycache_dirs` is assigned to but never used

Remove assignment to unused variable `pycache_dirs`

(F841)